sound/soc/sof/ipc4-topology.h

```diff
@@ -207,7 +207,7 @@ struct sof_ipc4_control_data {
 
 	union {
 		struct sof_ipc4_ctrl_value_chan chanv[0];
-		struct sof_abi_hdr data[0];
+		struct sof_ipc4_abi_hdr data[0];
```

Comment:
Shouldn't this belong to the `ASoC: SOF: create sof_ipc4_abi_hdr to support ipc4.` commit?

Reply:
OK. I will move it in that patch. Actually, I'm not sure which it is better. The patch ```ASoC: SOF: create sof_ipc4_abi_hdr to support ipc4``` modifies the uapi header file. Not sure modifying both uapi files and non-uapi file in one patch is a good idea or not.

Reply:
OK. I will move it in that patch. Actually, I'm not sure which it is better. The patch ```ASoC: SOF: create sof_ipc4_abi_hdr to support ipc4``` modifies the uapi header file. Not sure modifying both uapi files and non-uapi file in one patch is a good idea or not.